lib/smart_answer_flows/shared/minimum_wage_flow.rb

```diff
@@ -4,6 +4,7 @@ class MinimumWageFlow < Flow
       def define
         # Q1A
         multiple_choice :past_payment_date? do
+          option "2017-04-01"
```

Comment:
@ikennaokpala This option needs to be `2016-10-01`. `2017-04-01` won't be added until October 2017.